test/db/formats/elf/loongarch

```diff
@@ -0,0 +1,107 @@
+NAME=ELF: loongarch.elf
+FILE=bins/elf/analysis/loongarch64-curl
+ARGS=-A
+CMDS=?v entry0
+EXPECT=<<EOF
+0xa128
+EOF
+RUN
+
+NAME=ELF: loongarch.elf
+FILE=bins/elf/analysis/loongarch64-curl
+CMDS=<<EOF
+s sym.tool_seek_cb
+af
+e asm.sub.var=false
+pif
+EOF
+EXPECT=<<EOF
+ldptr.w  $a0, $a0, 0
```

Comment:
on mips i removed the $ and % sign, because thats att syntax. maybe you can do the same here honoring the asm.syntax